Dockerfile

```
@@ -0,0 +1,3 @@
+FROM gcr.io/distroless/static-debian12:nonroot
+COPY task /task
+CMD ["/task"]
```

Probably better if you would actually run this container for whatever reasons 😅
```suggestion
ENTRYPOINT ["/task"]
```

---

Since its based on distroless,  would you ever run it by itself though? I think this would maybe make sense with a usable base. Dunno